corehq/apps/saved_reports/models.py

```diff
@@ -818,20 +817,19 @@ def _export_report(self, emails, title):
             mock_request.GET = QueryDict('&'.join(mock_query_string_parts))
             request_data = vars(mock_request)
             request_data['couch_user'] = mock_request.couch_user.userID
-            if report_config.report_slug != ApplicationStatusReport.slug:
-                # ApplicationStatusReport doesn't have date filter
+            request_params = json_request(request_data['GET'])
+            if report_config.report_slug not in [ApplicationStatusReport.slug, CaseActivityReport.slug]:
+                # ApplicationStatusReport and CaseActivityReport don't have date filter
                 date_range = report_config.get_date_range()
-                start_date = datetime.strptime(date_range['startdate'], '%Y-%m-%d')
-                end_date = datetime.strptime(date_range['enddate'], '%Y-%m-%d')
-                datespan = DateSpan(start_date, end_date)
-                request_data['datespan'] = datespan
+                request_params['startdate'] = str(datetime.strptime(date_range['startdate'], '%Y-%m-%d'))
+                request_params['enddate'] = str(datetime.strptime(date_range['enddate'], '%Y-%m-%d'))
```

Comment:
Similar to before, calling `<datetime_object>.isoformat()` instead of `str(<datetime_object>)` would be more preferable.